DeckBuilder/Source/DeckBuilder.cpp

```diff
@@ -7,9 +7,51 @@ DeckBuilder::DeckBuilder()
     int maxNumberOfEachCard = 3;
 }
 
-void addCard(Card &cardToAdd)
+int DeckBuilder::checkNumberOfCard(Card card) 
 {
+    int number = 0;
+    for (Card c : deck.getCardsAsVector()) 
+    {
+        if (c == card) 
+        {
+            number++;
+        }
+    }
+    return number;
+}
+
+void DeckBuilder::addCard(Card &cardToAdd)
+{
+    int dl = deck.length();
```

Comment:
dazwij zmienna deckLength. Nic to nie kosztuje, a poprawia czytelnośc. I nie musze sie zastanawiać kilka sekund czym jest dl :D